packages/server/src/fhir/binary.test.ts

```diff
@@ -259,23 +259,16 @@ describe('Binary', () => {
       .set('Content-Type', ContentType.FHIR_JSON)
       .send({
         ...binary,
-        securityContext: { reference: 'Patient/123' },
```

Comment:
Why did we remove this test?

Can we instead add new test?

Reply:
If you expand the diff above, you'll see that this test case was a mis-named copy-paste and not really testing `securityContext` per se — I replaced it with a test of the changes I made.  The `securityContext` test case still exists and is being run